### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/locales/nl/app.json [hunk 1]
```diff
@@ -402,6 +402,55 @@
...
+              "text": "In samenwerking met BTC Direct, een vertrouwde Europese broker, biedt BitBox grootschalige over-the-counter (OTC) handel voor Bitcoin, Ethereum, stablecoins en meer met deskundige ondersteuning direct in de BitBoxApp.",
+              "title": "Koop en verkoop met de BTC Direct Private Trading Desk"
+            },
+            "paymentMethods": {
+              "buy": "Bitcoin kopen:",
+              "buy2": "Plaats grote kooporders bij je accountmanager, zet een prijs vast en betaal via bankoverschrijving. Je crypto wordt na betaling direct geleverd aan je BitBox.",
+              "fee": "Minimum handelsbedragen van €50.000 zijn van toepassing, met een transparante vergoeding tussen 0,5% en 1%.",
+              "sell": "Bitcoin verkopen:",
+              "sell2": "Zet een OTC-prijs vast wanneer je een verkooporder plaatst. De betaling van EUR vindt plaats zodra je je crypto hebt verstuurd naar BTC Direct. Persoonlijke assistentie is beschikbaar indien nodig.",
+              "title": "Betaalmethoden en kostendi"
```
Comment (Stadicus) [inline frontends/web/src/locales/nl/app.json:426]: s/kostendi/kosten/

**Review by Stadicus — CHANGES_REQUESTED** [review]
Small typo in Dutch

File: frontends/web/src/locales/nl/app.json (hunk 1 above)
Comment (benma) [reply frontends/web/src/locales/nl/app.json]: Fixed

File: frontends/web/src/locales/it/app.json [hunk 2]
```diff
@@ -1469,7 +1469,7 @@
         "La passphrase è un ulteriore livello di sicurezza oltre al backup.\n",
         "L'inserimento di una passphrase diversa genererà sempre un wallet diverso.",
         "Per ripristinare il wallet sono necessari <strong>sia la passphrase che il backup</strong>.",
-        "Se si dimentica la passphrase, <strong>non è più possibile accedere ai propri fondi</strong>.\n"
+        "Se si dimentica la passphrase, <strong>non è più possibile accedere ai propri fondi</strong>."
```
Comment (thisconnect) [inline frontends/web/src/locales/it/app.json:1472]: 👍

**Pull request merged**